driver/src/metrics/http_metrics.rs

```diff
@@ -0,0 +1,197 @@
+use anyhow::Result;
+use prometheus::{HistogramOpts, HistogramVec, Registry};
+use std::sync::Arc;
+use std::time::Duration;
+
+/// A registry for all HTTP related metrics.
+#[derive(Debug)]
+pub struct HttpMetrics {
+    latency: HistogramVec,
+    size: HistogramVec,
+}
+
+impl HttpMetrics {
+    /// Create a new HTTP metrics registry.
+    pub fn new(registry: &Arc<Registry>) -> Result<Self> {
+        let latency = HttpMetrics::initialize_histogram(
+            registry,
+            "dfusion_service_http_latency",
+            "Latency in seconds for HTTP request",
+            vec![0.05, 0.1, 0.25, 0.5, 1.0, 2.0, 5.0, 10.0],
```

Comment:
Why not [DEFAULT_BUCKETS](https://docs.rs/prometheus/0.3.2/prometheus/constant.DEFAULT_BUCKETS.html)?

Reply:
Missed that! Thanks!